src/AdvApi32.Desktop/AdvApi32.cs

```diff
@@ -507,10 +552,10 @@ public static partial class AdvApi32
         /// </returns>
         [DllImport(api_ms_win_security_base_l1_2_0, SetLastError = true)]
         [return: MarshalAs(UnmanagedType.Bool)]
-        public static extern unsafe bool GetTokenInformation(
+        public static extern bool GetTokenInformation(
             SafeObjectHandle TokenHandle,
             TOKEN_INFORMATION_CLASS TokenInformationClass,
-            void* TokenInformation,
+            IntPtr TokenInformation,
```

Comment:
I share the concern. The `IntPtr` overload should have already been constructed anyway.